src/actions/setIsMulticursorExecuting.ts

```diff
@@ -0,0 +1,29 @@
+import State from '../@types/State'
+import Thunk from '../@types/Thunk'
+
+interface SetIsMulticursorExecutingPayload {
+  value: boolean
+  /** Label for the undo operation (typically the command name). */
+  operationLabel?: string
+}
+
+/** Sets the isMulticursorExecuting flag. */
+const setIsMulticursorExecuting = (state: State, { value }: SetIsMulticursorExecutingPayload) => {
+  return {
+    ...state,
+    isMulticursorExecuting: value,
+  }
+}
+
+/** Action-creator for setIsMulticursorExecuting. */
+export const setIsMulticursorExecutingActionCreator =
+  (payload: SetIsMulticursorExecutingPayload): Thunk =>
+  dispatch => {
+    dispatch({
+      type: 'setIsMulticursorExecuting',
+      value: payload.value,
+      operationLabel: payload.operationLabel,
```

Comment:
You can use the spread operator instead of enumerating each payload property, as we do in other action-creators.

https://github.com/cybersemics/em/blob/ba2d8843010ccb39adfcb643b98b2f61e2503e47/src/actions/join.ts#L85-L89